sdk/src/test/scala/com/horizen/account/api/rpc/service/EthServiceTest.scala

```diff
@@ -173,11 +174,22 @@ class EthServiceTest extends JUnitSuite with MockitoSugar with ReceiptFixture wi
       }
       TestActor.KeepRunning
     })
-
     val nodeViewHolderRef: ActorRef = mockedSidechainNodeViewHolder.ref
+
     val transactionActorRef: ActorRef = SidechainTransactionActorRef(nodeViewHolderRef)
+
+    val mockedSyncStatusActor = TestProbe()
+    mockedSyncStatusActor.setAutoPilot((sender: ActorRef, msg: Any) => {
+      msg match {
+        case ReturnSyncStatus() =>
+          sender ! new SyncStatus(true, BigInt(250), BigInt(200), BigInt(300))
+      }
+      TestActor.KeepRunning
+    })
+    val mockedSyncStatusActorRef: ActorRef = mockedSyncStatusActor.ref
```

Comment:
why don't we use `mockedSyncStatusActorRef` from `AccountEthRpcRouteMock.scala`?
there we may configure to return status or throw an exception.

Reply:
I keep the Actor and ActorRef declaration in this test class because I did a refactor in the `AccountEthRpcRouteMock.scala`, there the `mockedSyncStatusActorRef` is just a ref without mocked logic because is used only for the `AccountEthRpcRouteTest.scala` that test the rpc interaction at higher level (manage of rpc batch, invalid json input, etc.)